_tuning-your-cluster/seperate-index-and-search-workloads.md

```diff
@@ -0,0 +1,171 @@
+
+By default, indexes created in a remote-store-enabled cluster use segment replication. For more information, see [Segment replication]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/segment-replication/index/).
+
+You can add search replicas for an index using the `number_of_search_replicas` setting (default is 0) in one the following ways.
+
+#### Option 1: Create an index with search replicas
+
+Use this option when you're creating a new index and want to configure search replicas from the start. This approach is ideal for planning your workload separation strategy before indexing data.
+
+The following request creates an index with one primary, one replica, and two search replicas:
+
+```json
+PUT /my-index
+{
+    "settings": {
+        "index": {
+            "number_of_shards": 1,
+            "number_of_replicas": 1,
+            "number_of_search_replicas": 2,
+        }
+  }
+}
+```
+{% include copy-curl.html %}
+
+#### Option 2: Update the search replica count for an existing index
+
+Use this option when you have an existing index and want to add or modify search replicas. This is useful when you need to adjust your search capacity based on changing workload demands.
+
+The following request updates the search replica count:
+
+```json
+PUT /my-index/_settings
+{
+  "settings": {
+    "index": {
+      "number_of_search_replicas": 1
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+#### Option 3: Restore an index from a snapshot with search replicas
+
+Use this option when you're restoring an index from a snapshot and want to configure search replicas during the restore process. This is particularly useful for disaster recovery scenarios or when migrating indexes between clusters.
+
+The following request restores an index from a snapshot with search replicas:
+
+```json
+POST /_snapshot/my-repository/my-snapshot/_restore
+{ 
+    "indices": "my-index", 
+    "index_settings": { 
+        "index.number_of_search_replicas": 2,
+        "index.replication.type": "SEGMENT"
+     } 
+}'
+```
+{% include copy-curl.html %}
+
+## Additional configuration
+
+After setting up basic workload separation, you can fine-tune your configuration to optimize performance and resource utilization. The following settings allow you to control search routing, automatically scale replicas, and manage write workloads based on your specific needs:
+
+### Enforce cluster-level search request routing
+
+When search replicas are enabled, all search traffic is routed to them by default. The following request enforces or relaxes this routing behavior:
+
+```json
+PUT /_cluster/settings
+{ 
+    "persistent": {
+        "cluster.routing.search_replica.strict": "true"
+    }
+}
+```
+{% include copy-curl.html %}
+
+- `true` (default): Route only to search replicas.
```

Comment:
These bullet points appear to be floating (not connected to anything). Should they be introduced by a brief sentence ending in a colon?

Reply:
I'll add that.